pytest/common.py

```diff
@@ -63,3 +64,8 @@ def stop_client(directory):
     assert ret.retval == 0
 
 
+def vmci_supported():
```

Comment:
The pytests will fail on a VMware VM (with VMCI) if `libzmq` does not support VMCI, because we only check if the VMCI device is present but not if VMCI ("vmci://...") is supported in `libzmq`.

I will fix that later.